### PR TITLE
Faq update (#12)

### PR DESCRIPTION
**Pull request by Jonnie-Dev**
* fix: add background to header

* fix: add margin to section-block

* feat: update faq section

* fix:changed  corporate header background overlay color

* fix: add background overlay to header

* feat: add margin to section-box

### CODE REVIEW

**Comment by saenuruki** [comment]
Please disable the navigation action. And also please remove the playback button.
https://github.com/MeltingHackTeam/CommunityWebsite/assets/9348579/7e9980c8-7d08-427a-bd77-1b56f79f73d3

**Comment by Jonnie-Dev** [comment]
Alright, on it

**Comment by Jonnie-Dev** [comment]
updated

File: public/images/faq.jpg
Comment (saenuruki) [inline public/images/faq.jpg:1]: Please remove it.

**Comment by saenuruki** [comment]
> Please disable the navigation action. And also please remove the playback button.

I think you haven't fixed this one.

**Comment by Jonnie-Dev** [comment]
Must have been an oversight, I'll fix it now. Thought I did already

**Comment by Jonnie-Dev** [comment]
Fixed

**Review by saenuruki — APPROVED** [review]

**Pull request merged**